src/components/heatmap/utils.js

```diff
@@ -37,12 +37,15 @@ export function onHeatmapMouseMove(event, props) {
   if (cellId) {
     // Use the cell ID to look up the cell information object.
     const cellInfo = cells[cellId];
-    updateCellsHover({
-      cellId,
-      mappings: { xy: cellInfo.xy, ...cellInfo.mappings },
-      uuid,
-      factors: cellInfo.factors,
-    });
-    updateStatus(makeCellStatusMessage(cellInfo.factors));
+    const { factors = {}, xy, mappings = {} } = cellInfo || {};
+    if (cellInfo) {
```

Comment:
We're essentially doing the same truth check on `cellInfo` on this line and the one above... Could the assignment just be moved inside the `if`, and remove the `|| {}`?

(Also, do we currently have cases where cellInfo will be null, or this just to be safe? I might take that out, unless we know we have that condition now, and it's really ok.)

Reply:
It was just to be safe, but it makes sense that we may want things to fail to catch bugs